cmd/root.go

```diff
@@ -12,14 +12,15 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var openEVEC *openevec.OpenEVEC
```

Comment:
Do you think it makes sense to pass this object during the construction of CLI?

Reply:
Actually we need `configName` to initialize it properly (with particular config), so I do it in `PersistentPreRunE` where we have it.